saleor/webhook/observability/obfuscation.py

```diff
@@ -201,6 +200,9 @@ def anonymize_event_payload(
 ) -> Any:
     if not subscription_query:
         return payload
+
+    from ...graphql.api import schema
```

Comment:
Doesn't this solution generate additional memory consumption for each query for enabled observability?

Reply:
I will measure the impact with Memray.